src/Vizkit3dWorld.cpp

```diff
@@ -20,6 +20,9 @@
 
 namespace vizkit3d_world {
 
+static int QT_ARGC = 1;
+static char* QT_ARGV[1] = { "vizkit3d_world" };
```

Comment:
warning: ISO C++ forbids converting a string constant to ‘char*’

```suggestion
static char* QT_ARGV[1] = {(char *)"vizkit3d_world"};
```